styles/spotify-web/catppuccin.user.css

```diff
@@ -0,0 +1,395 @@
+      * {
+      color: @crust !important;
+      fill: @crust !important;
+    }
+    .bk509U3ZhZc9YBJAmoPB {
+      background: @mantle;
+    }
+    .HVCCFeUiHVwZVv74p34a * {
+      background: @surface0 !important;
+    }
+    .QO9loc33XC50mMRUCIvf {
+      background-color: @surface0;
+    }
+    .QO9loc33XC50mMRUCIvf:focus {
+      -webkit-box-shadow: 0 0 0 2px @text;
+      box-shadow: 0 0 0 2px @text;
+    }
+    .QO9loc33XC50mMRUCIvf:hover {
+      background-color: @surface1;
+    }
+    .H6jh9Xd7DNOq3NsLDmCB:active,
+    .H6jh9Xd7DNOq3NsLDmCB:focus,
+    .H6jh9Xd7DNOq3NsLDmCB:hover {
+      color: @text !important;
+    }
+    .H6jh9Xd7DNOq3NsLDmCB {
+      color: @text;
+    }
+    .kPpCsU {
+      fill: @text;
+    }
+    .VgSbatGBB9XwTH2_dsxg .ql0zZd7giPXSnPg75NR0 {
+      background: @base !important;
+      color: @text;
+    }
+    .HsbczDqu9qjcYr7EIdHR,
+    .rovbQsmAS_mwvpKHaVhQ * {
+      background: transparent !important;
+    }
+    .Z35BWOA10YGn5uc9YgAp,
+    .pQmF4tvRpUeLWgPKUcW7 {
+      background-color: @crust;
+    }
+    // Explicit icon
+    .kpGMQq1KFz620g_BD_dS {
+      background-color: @overlay0;
+    }
+    .link-subtle {
+      color: @subtext1;
+      &:hover {
+        color: @text;
+      }
+    }
+    // Search cards
+    .LunqxlFIupJw_Dkx6mNx {
+      background: @mantle !important;
+    }
+    .bQthUEx0_U98DJkT1saO,
+    .RVRoa p {
+      color: @text;
+    }
+    // Social buttons
+    .kmZeYl:hover {
+      background-color: @crust;
+    }
+    .kmZeYl {
+      background-color: @mantle;
+    }
+    // Icon
+    .RfidWIoz8FON2WhFoItU {
+      color: @text;
+    }
+    // Playlist bottom
+    .cuLHaM {
+      background-color: @base;
+    }
+
+    // Sidebar top
+    .y2UicQnlTq148rL8Y0jp {
+      box-shadow: 0 6px 10px @mantle;
+    }
+    // Bottom player
+    .vnCew8qzJq3cVGlYFXRI {
+      background-color: @text;
+    }
+    .vnCew8qzJq3cVGlYFXRI * {
+      fill: @crust;
+    }
+
+    .rovbQsmAS_mwvpKHaVhQ .PFgcCoJSWC3KjhZxHDYH * {
+      fill: @text !important;
+    }
+    .TywOcKZEqNynWecCiATc {
+      --bg-color: @surface1;
+      --fg-color: @text;
+      --is-active-fg-color: @accent-color;
+    }
+    a {
+      color: @accent-color;
+    }
+    .Ng3dPPA2_1CFYkzPukjM {
+      background: @blue;
+    }
+    .KAZD28usA1vPz5GVpm63.EHxL6K_6WWDlTCZP6x5w::after {
+      background-color: @accent-color;
+    }
+  }
+  /*  Application of the Theme
+   *  The values below are examples and may need to be adapted.
+   *  To apply the correct Theming call the above defined Mixin and pass
+   *  your desired Flavor and Accent Color to it. These are usually
+   *  populated through the `@var`s in the Meta Block. */
+  // synced colorscheme
+
+  @media (prefers-color-scheme: light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  @media (prefers-color-scheme: dark) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+} /* /@-moz-document */
```

Comment:
You can remove this `/* /@-moz-document */` comment at the end, it was a mistake in the template I think.